### PR TITLE
Updates to go live

### PR DESCRIPTION
**Pull request by KatherineMichel**
Changes proposed in this PR:
- Fixing number of dates/dates
- Removing tutorial references
- Removing attendance reference

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -5,11 +5,11 @@
 <header class="header">
   <div class="row">
     <div class="header-text">
-      <h1>Join us for three days of talks and tutorials</h1>
+      <h1>Join us for two days of talks</h1>
 
       <div class="event-info-detail">
         <strong
-          class="event-info-date">October 21-23, 2021</strong>
+          class="event-info-date">October 22-23, 2021</strong>
```
[the remark this answers is not in the document]
Comment (KatherineMichel) [reply _layouts/home.html:12]: I don't know. We usually have tutorials first. That's why I put it that way. Either way is fine for me personally.

[the remark this answers is not in the document]
Comment (drewbrew) [reply _layouts/home.html:12]: We decided on Friday/Saturday, just to close this discussion for someone looking at this in the future.

**Comment by dawnwages** [comment]
All done with sponsors stuff! thanks for your patience yall. @KatherineMichel @katialira @jefftriplett

File: _includes/sponsor-footer.html [hunk 2]
```diff
@@ -1,5 +1,11 @@
 <div class="partner-footer section-pad">
-  <h3 class="v-pad-bottom text-center">Our Sponsors</h3>
+  <h3 class="v-pad-bottom text-center">Sponsors</h3>
+      <div class="row v-pad-top">
+        <div class="medium-6 column medium-centered text-center">
+        <p>Interested in becoming a sponsor? There's still time!</p>
+        <a href="/sponsors/information/" class="button">Become a Sponsor</a>
+      </div>
+        </div>
```
Comment (drewbrew) [inline _includes/sponsor-footer.html:8]:
```suggestion
    <div class="row v-pad-top">
      <div class="medium-6 column medium-centered text-center">
        <p>Interested in becoming a sponsor? There's still time!</p>
        <a href="/sponsors/information/" class="button">Become a Sponsor</a>
      </div>
    </div>
```

Minor indentation fix here

File: _pages/sponsors-information.md [hunk 3]
```diff
@@ -26,33 +26,54 @@ Because the conference is organized by [DEFNA](https://www.defna.org/), a non-pr
...
+| Sponsor link on website                                                            | :heavy_check_mark:  |                     |                    |
+
+#### Speech-to-Text Sponsor $3,500 (Limit 1)
+
+In order to prioritize a comfortable and accessible conference, we transcribe all presentations to ensure all potential viewers can visually access audio information through in-sync captioning. Your name and logo will be mentioned in social media posts, as a sponsor link on website, and featured in conference videos.
+
+
+#### Virtual Conference Hall Sponsor $2,500 (Limit 1)
+
+Due to the fully remote conference this year, we'll be working with [spatial.chat](Spatial.chat) to imitate the real conference experience for the one day of talks. Your name and logo will be mentioned in social media posts, as a sponsor link on website, and featured in conference videos. If you're familiar with the Pycascades 2021 conference, it's the same platform! [See spacial chat demo >>](https://www.youtube.com/watch?v=EVo1REyNa3Y)
```
Comment (drewbrew) [inline _pages/sponsors-information.md:63]:
```suggestion
Due to the fully remote conference this year, we'll be working with [spatial.chat](Spatial.chat) to imitate the real conference experience for the Friday session. Your name and logo will be mentioned in social media posts, featured as a sponsor link on our website, and shown in conference videos. If you're familiar with the PyCascades 2021 conference, it's the same platform! [See Spatial Chat demo >>](https://www.youtube.com/watch?v=EVo1REyNa3Y)
```

File: _pages/sponsors-information.md [hunk 4]
```diff
@@ -26,33 +26,54 @@ Because the conference is organized by [DEFNA](https://www.defna.org/), a non-pr
...
+
+#### Invitation Sponsor $2,000 (Limit 1)
+
+All registrants receive an invitation to join the conference platform, directing them how to sign up for the Discord and thanking them for their attendance. Your logo will be featured at the bottom of this email, welcoming attendees to the event! You will receive name and logo on video stream interstitial, sponsor link on website, and your name and logo on the email to participants to register.
```
Comment (drewbrew) [inline _pages/sponsors-information.md:67]:
```suggestion
All registrants receive an invitation to join the conference platform, directing them how to sign up for the Discord and thanking them for their attendance. Your logo will be featured at the bottom of this email, welcoming attendees to the event! You will receive your name and logo on the video stream interstitials, a sponsor link on the DCUS website, and your name and logo on the email to participants confirming their registration.
```

File: _pages/sponsors-information.md [hunk 5]
```diff
@@ -26,33 +26,54 @@ Because the conference is organized by [DEFNA](https://www.defna.org/), a non-pr
...
+
+#### Speaker Equipment Sponsor $1,500 (Limit 3)
+
+All speakers who request help with equipment will receive assistance from our Audio Visual team. Sometimes this equipment can be costly. With your contribution, you will help our team provide top tier content without burdening the resources of our amazing speakers. You will receive name and logo on video stream interstitial, sponsor link on website and your name and logo on social media post, thanking you for your contribution.
```
Comment (drewbrew) [inline _pages/sponsors-information.md:71]:
```suggestion
All speakers who request help with equipment will receive assistance from our Audio/Visual team. Sometimes this equipment can be costly. With your contribution, you will help our team provide top tier content without burdening the resources of our amazing speakers. You will receive your name and logo on the video stream interstitials, a sponsor link on the DCUS website and your name and logo on a social media post thanking you for your contribution.
```

File: _pages/sponsors-information.md [hunk 6]
```diff
@@ -26,33 +26,54 @@ Because the conference is organized by [DEFNA](https://www.defna.org/), a non-pr
...
+
+#### Diversity Sponsor
+
+Django and the Python community at large pride themselves on their commitment to diversity and inclusion. Talk to us about contributing to our Opportunity Grant Fund, which prioritizes grants for people from underrepresented groups. Diversity sponsorship funds will also be used to offset the cost of offering child care!
```
Comment (drewbrew) [inline _pages/sponsors-information.md:75]:
```suggestion
Django and the Python community at large pride themselves on their commitment to diversity and inclusion. Talk to us about contributing to our Opportunity Grant Fund, which prioritizes grants for people from underrepresented groups. Diversity sponsorship funds will also be used to offset the cost of offering child care when we return to in-person conferences!
```

**Review by drewbrew — COMMENTED** [review]
Just some copy edit suggestions here

**Pull request merged**